src/score_gen/score.rs

```diff
@@ -298,7 +299,10 @@ mod tests {
         let mut rng = rand::thread_rng();
 
         let secret_k = BlsScalar::random(&mut rng);
-        let pk_r = PublicSpendKey::from(SecretSpendKey::default());
+        let pk_r = PublicSpendKey::from(SecretSpendKey::new(
```

Comment:
Same as above; no need to manually derive keys when you have a random number generator available